### ISSUE TITLE
Inaccurate physical disk latencies from Perflib

### ISSUE THREAD

**Issue by cbwest3** [prometheus-community/windows_exporter#1154]
**TLDR; Perfmon, [`fio`](https://github.com/axboe/fio), and [PDH](https://learn.microsoft.com/en-us/windows/win32/api/pdh/)-based metrics report the same physical disk latency, Perflib-based metrics do not.**

I'm trying to add a `physical_disk` collector  and found that latencies are incorrect when using the Perflib functionality native to [`windows_exporter`](https://github.com/prometheus-community/windows_exporter). I wrote two `physical_disk` collectors, one based on Perflib and one based on [Performance Data Helper (PDH)](https://learn.microsoft.com/en-us/windows/win32/api/pdh/), and ran them side-by-side.

The screenshot below shows the [`fio`](https://github.com/axboe/fio) command used to run the test. You can also see the reported latency in the test summary (8.23ms).

![fio-output](https://user-images.githubusercontent.com/8591862/224769684-b47086eb-b9de-4dd5-9906-4a7fd2a7a974.png)

The following graph shows the Perfmon chart from the test. The Permon latencies agree with fio above (0.009 seconds, ~9 milliseconds).

![perfmon](https://user-images.githubusercontent.com/8591862/224769914-538a9b85-0d64-4977-a219-23944a8c4cfa.png)

Lastly, the following Grafana plot shows `windows_exporter` `physical_disk` latencies implemented for both Perflib and PDH in https://github.com/cbwest3/windows_exporter/releases/tag/comparison_testing. This plot shows that PDH-based latencies agree with Perfmon and `fio` (just under 10ms), while Perflib-based latencies do not (983 ms).

![grafana-pdh-vs-perflib](https://user-images.githubusercontent.com/8591862/224770277-ac910e30-8b52-41bb-bd1c-2f08527fe778.png)

**Comment by rob-scheepens** [comment]
Based on this resolution I've created PR https://github.com/prometheus-community/windows_exporter/pull/1211.
